assignment/tool/src/webapp/vm/assignment/instructor_new_edit/assignto_section.vm

```diff
@@ -53,8 +53,8 @@ can be either all members of the site, all members of selected groups, or the se
 			#set($hideGB = "if (document.getElementById('addToGradebook') != null){document.getElementById('addToGradebook').style.display = 'none';document.getElementById('$!name_Addtogradebook').checked=false};")
 		#end
 		<input type="radio" name="$name_AssignTo" id="groups" value="$value_AssignTo_IndividualsFromGroups"
-			onclick="$hideGB$('#groupTableContainer').fadeIn('slow'); ASN_INE.validateGroupSelection();ASN.resizeFrame(); if($!isTimesheet){ASN.enableTimesheetSetupSection();}"
-			#if( $value_AssignTo == $value_AssignTo_IndividualsFromGroups && !$selectedGroupsNotFound ) checked="checked"
+			onclick="$hideGB$('#groupTableContainer').fadeIn('slow'); ASN_INE.validateGroupSelection();ASN_INE.handleAssignToChangeForPeerAssessment();ASN.resizeFrame(); if($!isTimesheet){ASN.enableTimesheetSetupSection();}"
```

Comment:
ASN_INE.handleAssignToChangeForPeerAssessment()  Where does this come from?

Reply:
I don't know where i think that is an older function, i will remove it

Reply:
Please, could you test this without that ? I don't think it belongs to the functionality.

Reply:
I have tested it and works without that